app/preferences/styles.ts

```diff
@@ -0,0 +1,108 @@
+'use client';
+
+import NextImage from 'next/image';
+import styled from 'styled-components';
+import COLORS from '../../styles/colors';
+
+export const Background = styled.main`
+  flex-direction: column;
+  min-width: 100%;
+  min-height: 100svh;
+  display: flex;
+  align-items: center;
+  justify-content: start;
+  background-color: ${COLORS.gray2};
+  overflow: hidden;
+`;
+
+export const InlineContainer = styled.main`
+  width: 30%;
+  height: 85%;
+  flex-direction: column;
+  margin-top: 2%;
+  margin-bottom: 2%;
+
+  @media (max-width: 1200px) {
+    width: 45%;
+  }
+  @media (max-width: 768px) {
+    width: 85%;
+  }
+`;
+
+export const Image = styled(NextImage)`
+  width: 20px;
+  height: 20px;
+  margin-bottom: -2px;
+`;
+
+export const Rectangle = styled.main<{
+  variant: 'light' | 'dark';
+  widthPercentage: string;
+}>`
+  width: ${({ widthPercentage }) => widthPercentage};
+  height: 7px;
+  display: inline-block;
+  background: ${({ variant }) =>
+    variant === 'light' ? COLORS.gray8 : COLORS.gray10};
+  border-radius: 2px;
+`;
+
+export const Container = styled.main`
+  width: 100%;
+  height: 53%;
+  display: flex;
+  flex-direction: column;
+  align-items: start;
+  margin: 25px 0px;
+  justify-content: space-between;
+  background-color: white;
+  border-radius: 8px;
+  padding: 13%;
+`;
+
+export const InputText = styled.text`
+  margin-top: 6%;
+`;
+
+export const Input = styled.input`
+  width: 100%;
+  height: 22px;
+  border-color: ${COLORS.gray4};
+  border-style: solid;
+  border-radius: 4px;
```

Comment:
```
margin-bottom: 1.75rem;
margin-top: 0.25rem;
padding-left: 0.4rem;
```
for font-size, padding, and margin, it is good practice to use `rem` or `em` since they help create responsive designs! `rem` is relative to the root (`html`) element's font size and `em` is relative to the parent element's font size. i added `margin-top` to ensure that we have more space between the text and the input box. `padding-left` helps add space between the cursor and the left side of the container in your `Input` component. i recommend you read over the styling guide in our b&r notion page, and if you have any questions, please let me know!